cli/lib/kontena/cli/stacks/yaml/reader.rb

```diff
@@ -319,13 +368,12 @@ def interpolate(content, use_opto: true, substitutions: {}, raise_on_unknown: fa
               if use_opto
                 opt = variables.option(var)
                 if opt.nil?
-                  if variables.find { |opt| opt.to[:env][var] }
```

Comment:
This seems like a regression, with a `to: env: FOO` variable, you can no longer interpolate `${FOO}`:

```
 [error] RuntimeError : Undeclared variable 'FOO' in /home/kontena/kontena/kontena/cli/test/from-variable.yaml:16 --       - FOO=${FOO}
```